appveyor.yml

```diff
@@ -26,6 +33,8 @@ install:
 
 build_script:
   - c:\gopath\bin\dep.exe ensure
+  - go build -o torus.exe
```

Comment:
Hmm, we're probably going to have to separate out building the windows binary (and now the installer + uploading it) from the rest of the Makefile when we build a release through `make release-all`.

If you want to introduce a `make msi` that requires the os to be windows and for `wix/go-msi` to be available (if possible) then I think we can open another issue (or subsequent PR) that has the changes to actually separately build, package, and upload a windows release.

Reply:
I introduced an initial `make msi` that checks for `go-msi`, but I'll add os detection and hook it into the appveyor.yml.